virtual_rainforest/models/animals/constants.py

```diff
@@ -9,39 +9,51 @@
 avoid frequent searches through this constants file for values.
 """  # noqa: D205, D415
 
+from dataclasses import dataclass, field
+
 from virtual_rainforest.models.animals.animal_traits import (
     DietType,
     MetabolicType,
     TaxaType,
 )
 
-ENDOTHERMIC_METABOLIC_RATE_TERMS: dict[TaxaType, tuple[float, float]] = {
-    TaxaType.MAMMAL: (0.75, 0.018),
-    # Mammalian metabolic rate scaling from Metabolic Ecology p210. [assumes g mass]
-    TaxaType.BIRD: (0.75, 0.05),
-    # Toy values.
-}
 
-ECTOTHERMIC_METABOLIC_RATE_TERMS: dict[TaxaType, tuple[float, float]] = {
-    TaxaType.INSECT: (0.75, 0.08)
-    # Insect metabolic rate scaling from Metabolic Ecology p210. [assumes g mass]
-}
+@dataclass(frozen=True)
+class AnimalConsts:
+    """Dataclass to store all constants related to metabolic rates.
+
+    TODO: The entire constants fille will be reworked in this style after the energy to
+    mass conversion.
 
-METABOLIC_RATE_TERMS: dict[MetabolicType, dict[TaxaType, tuple[float, float]]] = {
+    """
+
+    metabolic_rate_terms: dict[MetabolicType, dict[str, tuple[float, float]]] = field(
+        default_factory=lambda: {
+            # Parameters from Madingley, mass-based metabolic rates
+            MetabolicType.ENDOTHERMIC: {
+                "basal": (4.19 * 10**10, 0.69),
+                "field": (9.08 * 10**11, 0.7),
+            },
+            MetabolicType.ECTOTHERMIC: {
+                "basal": (4.19 * 10**10, 0.69),
+                "field": (1.49 * 10**11, 0.88),
+            },
```

Comment:
You can write scientific notation like this:

```suggestion
            MetabolicType.ENDOTHERMIC: {
                "basal": (4.19e10, 0.69),
                "field": (9.08e11, 0.7),
            },
            MetabolicType.ECTOTHERMIC: {
                "basal": (4.19e10, 0.69),
                "field": (1.49e11, 0.88),
            },
```

I am wondering whether these would be better as dataclasses rather than dicts.

Reply:
I am doing a constants rework immediately after this PR to avoid the PR sprawling out into the whole animal module.